### PR TITLE
README.rst: Add examples and how-to's for setup

### PR DESCRIPTION
**Pull request by RJ722**
Add examples output for Vulture.

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation in the README file.
- [x] I have updated the documentation accordingly.
- [ ] I have added an entry in NEWS.rst.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by RJ722** [comment]
Would it be a nice idea to also mention little snippets of `.travis.yml` or `appveyor.yml` showing how to add Vulture to CI tests?

File: tox.ini [hunk 1]
```diff
@@ -16,7 +16,9 @@ commands =
   py.test
 
 [testenv:docs]
-deps = collective.checkdocs
+deps =
+    collective.checkdocs
+    pygments
```
Comment (jendrikseipp) [inline tox.ini]: Why do we need pygments?

**Comment by jendrikseipp** [comment]
Thanks for looking into this! I think instead of snippets for CI, we should document the exit codes. Let me make some changes to the patch.

File: tox.ini (hunk 1 above)
Comment (RJ722) [reply tox.ini]: The `pygments` library is required for syntax highlighting.

Comment (jendrikseipp) [reply tox.ini]: Ah, I see, thanks!

**Comment by RJ722** [comment]
> we should document the exit codes

Yes, that is a great idea. I'm gonna make a PR.

**Pull request merged**